src/client/components/forms/relationship.js

```diff
@@ -34,7 +34,8 @@ const validators = require('../../helpers/react-validators');
 const RelationshipRow = require('./parts/relationship-row');
 
 /**
- * Checks if the relationship is new by checking that the fields weren't set when the form was loaded
+ * Checks if the relationship is new by checking that
+   the fields weren't set when the form was loaded
```

Comment:
Missing asterisk `*` at the start here, and on the other lines you've created in this commit